tests/pytests/test_vecsim.py

```diff
@@ -1142,7 +1139,7 @@ def test_wrong_vector_size():
     conn = getConnectionByEnv(env)
     dimension = 128
 
-    for data_type in VECSIM_DATA_TYPES:
+    for data_type in VECSIM_DATA_TYPES_EXTENDED:
```

Comment:
i suggest to add a query with wrong dimension to this test

Reply:
I added a generic validation for each type in `test_search_errors`